### ISSUE TITLE
[a11y][suggest] 'No suggestions' is not announced to the screen reader when displayed on the screen

### ISSUE THREAD

**Issue by jjaeggli** [microsoft/vscode#188705]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: Monaco v0.40.0-dev.20230626
- OS Version: Mac OS 13.4.1 (22F82), Windows 10 Enterprise 10.0.19045 Build 19045, Jaws 2020.2006.12 ILM, NVDA 2022.3

Steps to Reproduce:

1. Enable a screen reader such as VoiceOver or NVDA
1. Open the monaco interactive playground https://microsoft.github.io/monaco-editor/playground.html
2. Type some characters which will produce an unmatched prefix, ie. `zz`
3. Type ctrl+space to open autocomplete popup
4. Notice that the screen reader does not announce the "No suggestions." text in the listbox. The user cannot navigate to highlight the item when the listbox is open.

![Screenshot 2023-07-24 at 12 19 18 PM](https://github.com/microsoft/vscode/assets/13710580/ef086d3f-1d0e-4a7c-961c-cb83b9998d22)

This seems to be a violation of [WCAG 2.1 4.1.3](https://www.w3.org/WAI/WCAG21/Understanding/status-messages.html)

**Comment by VSCodeTriageBot** [comment]
Thanks for creating this issue! It looks like you may be using an old version of VS Code, the latest stable release is 1.80.1. Please try upgrading to the latest version and checking whether this issue remains.

Happy Coding!

**Comment by jjaeggli** [comment]
This should be resolved by the associated PR.
